src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlStyle.java

```diff
@@ -531,51 +531,77 @@
     // The following constants are used for the components of a signature of an element
```

Comment:
Two lines earlier, line 529, change `member signature` to `signatures`

Reply:
Good catch...